src/Calendar/MonthDropdown.tsx

```diff
@@ -61,7 +65,7 @@ class MonthDropdown extends React.PureComponent<MonthDropdownProps> {
 
   getRowCount = () => {
     const { limitEndYear } = this.props;
-    return getYear(new Date()) + limitEndYear;
+    return getYear(legacyParse(new Date())) + limitEndYear;
```

Comment:
If the argument of `legacyParse` is a Date type, not a string, is it unnecessary to use the` legacyParse` function?